src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.IntegrationTests/Microsoft.Azure.Services.AppAuthentication.IntegrationTests.csproj

```diff
@@ -0,0 +1,69 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  
+	<PropertyGroup>
+    
+		<TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+  
+		<SignAssembly>True</SignAssembly>
```

Comment:
@varunsh-msft you  do not need this property, it's already been added as part of common properties references for the entire repo